db/db_impl/db_impl.cc

```diff
@@ -1880,6 +1880,16 @@ std::vector<Status> DBImpl::MultiGet(
   }
 #endif  // NDEBUG
 
+  if (tracer_) {
```

Comment:
There are a few different MultiGet overloaded APIs. We should trace all of them.